README.md

```diff
@@ -1 +1,118 @@
 # Team-A-Agile
+
+## Table of Contents
+- Setup Instructions
+    - Backend Setup
+        - Java SDK
+        - Intellij
+    - Frontend Setup
+        - Microsoft Visual Studio
+- Testing Instructions
+    - Unit Tests
+    - Cucumber Tests
+
+## Setup Instructions
+
+### Backend Setup
+
+####  **Java SDK**
+This project requires Java JDK version 16:
+- [JDK 16 Download](https://www.oracle.com/java/technologies/javase-jdk16-downloads.html)
+
+
+####  **Intellij**
+For the backend you will need to install Intellij Community Edition if you don't already have it. Here is a link to the download page:
+
+- [Intellij Download](https://www.jetbrains.com/idea/download/#section=mac)
+
+You will then need to open the JavaWebService file in Intellij which is in:
+
+ Team-A-Agile/backend
+
+Once the project is opened the pom.xml file will run and install the necessary dependencies. If it doesn't then open this file in the editor and reload it by clicking the cycling arrow on the top right.
+
+You will need to create a file called "config.properties" in the resource folder to hold your database credentials. The resource folder is located in:
+
+src/main
```

Comment:
```suggestion
src/main/resources
```

Reply:
that path is referring to where the resources file is